openapi-bal-service/src/main/java/io/ballerina/openapi/converter/utils/ServiceToOpenAPIConverterUtils.java

```diff
@@ -159,38 +152,53 @@ private static void extractListenersAndServiceNodes(String serviceName, List<Str
                 endpoints.add(listener);
             }
             if (syntaxKind.equals(SyntaxKind.SERVICE_DECLARATION)) {
-                /**
-                 * Here check the service is related to the http module by checking listener type that attached to
-                 * service endpoints.
+                ServiceDeclarationNode serviceNode = (ServiceDeclarationNode) node;
+                /*
+                  Here check the service is related to the http module by checking listener type that attached to
+                  service endpoints.
                  */
```

Comment:
Let's use end-of-line comments within the method bodies. Please correct other relevant places as well.